README.md

```diff
@@ -1,368 +1,257 @@
++ `containers`: "smart components" (components that have functionality and logic that is passed to child "dumb components").
++ `components`: includes "dumb components", or components that simply render their properties and call functions given to them as properties by a parent component. Ultimately, at least one of these dumb components will have a parent container component.
 
-The `server_bundle_js_file` must correspond to the bundle you want to use for server rendering.
+You may also notice the `app/lib` folder. This is for any code that is common between bundles and therefore needs to be shared (for example, middleware). 
 
-```ruby
-# Shown below are the defaults for configuration
-ReactOnRails.configure do |config|
-  # Client bundles are configured in application.js
-  # Server bundle is a single file for all server rendering of components.
-  config.server_bundle_js_file = "app/assets/javascripts/generated/server.js" # This is the default
+#### Additional Redux Folders
+If you have used the `--redux` generator option, you will notice the familiar additional redux folders in addition to the aforementioned folders. In this organization paradigm, each bundle has its own store. We do not set a global store and then use partial stores based off of that. Again, this is for bundle code modularity and isolation. 
 
-  # Below options can be overriden by passing to the helper method.
-  config.prerender = false # default is false
-  config.generator_function = false # default is false, meaning that you expose ReactComponents directly
-  config.trace = Rails.env.development? # default is true for development, off otherwise
+### Using Images and Fonts
+The generator has created two symbolic links in your `client/app` folder that point to the fonts and images folders in Rails (meaning the ones inside of `app/assets/`). All of your fonts and images should still be kept in those folders. They will be visible for you to reference from your client application if you choose to do so.
 
-  # For server rendering. This can be set to false so that server side messages are discarded.
-  config.replay_console = true # Default is true. Be cautious about turning this off.
-  config.logging_on_server = true # Default is true. Logs server rendering messags to Rails.logger.info
-  
-  # Settings for the pool of renderers:
-  config.server_renderer_pool_size  ||= 1  # ExecJS doesn't allow more than one on MRI
-  config.server_renderer_timeout    ||= 20 # seconds
-end
-```
+### Bootstrap Integration
+React on Rails ships with Twitter Bootstrap already integrated into the build. Note that the generator removes `require_tree` in both the application.js and application.css.scss files. This is to ensure the correct load order for the bootstrap integration, and is usually a good idea in general. You will therefore need to explicitly require your files.
```

Comment:
I feel that if we're doing bootstrap, then we should have some minimal use of react-bootstrap as well.

this can be for the next PR.